twoopstracker/twoops/views.py

```diff
@@ -257,7 +259,7 @@ def get_queryset(self):
                 search_type = get_search_type(query)
                 if search_type == "raw":
                     query = refromat_search_string(query)
```

Comment:
```suggestion
```
We don't need this as well... Instead of `one, two, three`, users will type `one OR two OR three`